### DISCUSSION TITLE
✨Aug 2022 Community Spotlight 🎊 - @p0p4k and @neurlang

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#1836] [Announcements]
## TTS Contributor Spotlight: @p0p4k and @neurlang

![image](https://user-images.githubusercontent.com/1402048/183289605-d600932c-006c-48f8-b757-0570466c72fc.png)

Open source often isn’t as glamorous as it’s made out to be on TV. (Is it ever made out to be glamorous anywhere?) A lot, if not most, of the work, is behind the scenes, e.g. writing documentation, answering community questions, fixing bugs…perfecting and polishing things seldom seen. This is where the real work happens. So, this month we wanted to shine the contributor spotlight on [p0p4k](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=c99420f10c&e=f854fa5bd1) and [neurlang](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=911619d6cd&e=f854fa5bd1) who perfect and polish things seldom seen.

They both help a lot with the community, both in [GitHub Discussions](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=a2712f96cd&e=f854fa5bd1) and in our [Matrix Chat](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=aeffbc4485&e=f854fa5bd1). They also send awesome ✨PRs✨fixing issues both in the 🐸[TTS](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=2380e97580&e=f854fa5bd1) and 👟[Trainer](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=d65df258b2&e=f854fa5bd1) repos. As a plus, also check out @neurlang Japanese 🐸[TTS model](https://coqui.us1.list-manage.com/track/click?u=6faea2f1f19c814566173ffb0&id=738a71e1cd&e=f854fa5bd1).

**Comment by SiameseJuly** [comment]
您好，邮件我已收到。
